js/adapt-contrib-resources.js

```diff
@@ -2,7 +2,12 @@ import Adapt from 'core/js/adapt';
 import ResourcesView from './ResourcesView';
 import ResourcesHelpers from './ResourcesHelpers';
 
-class Resources {
+class Resources extends Backbone.Controller {
+  
+  initialize() {
+    this.listenTo(Adapt, 'adapt:start', this.initResources); 
```

Comment:
Could you also change 
` Adapt.on('resources:showResources', () => {`
to:
`this.listenTo(Adapt, 'resources:showResources', () => {`